tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -0,0 +1,836 @@
+            namespace_dhcp: dict[str, list[object]] = {}
+            for d in cmd.dhcp:
+                lst = namespace_dhcp.get(cast(str, d.namespaceName))
+                if not lst:
+                    lst = []
+                    namespace_dhcp[cast(str, d.namespaceName)] = lst
+                lst.append(cast(object, d))
+
+            @mevoco.in_bash
+            @mevoco.lock.file_lock('/run/xtables.lock')
+            def _remove_ebtable_rules_for_vfnics(dhcpInfo: object) -> None:
+                DHCPNAMESPACE = dhcpInfo.namespaceName
```

Ruff (0.15.2)

[error] Local variable `DHCPNAMESPACE` is assigned to but never used

Remove assignment to unused variable `DHCPNAMESPACE`

(F841)